internal/testutil/test_setup.go

```diff
@@ -60,7 +62,10 @@ func cleanupFakeCliFile(c *config.Config) {
 	}
 	if stat.Size() < 1000 {
 		// this is a fake CLI, removing it
-		_ = os.Remove(c.CliSettings().Path())
+		err = os.Remove(c.CliSettings().Path())
+		if err != nil {
+			log.Warn().Err(err).Msg("Failed to remove fake CLI")
```

Comment:
I wouldn't add this - Had that previously and yielded in a lot of error messages.

Reply:
The condition was flipped (`==` instead of `!=`). This commit fixes it